packages/tracing/src/browser/request.ts

```diff
@@ -41,20 +41,24 @@ export interface RequestInstrumentationOptions {
 /** Data returned from fetch callback */
 export interface FetchData {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  args: any[];
+  args: any[]; // the arguments passed to the fetch call itself
   fetchData?: {
     method: string;
     url: string;
     // span_id
     __span?: string;
   };
-  response?: Response;
+
+  // TODO (kmclb) Once type fix goes through, turn this back into a Response
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  response?: any;
```

Comment:
Maybe let's use `unknown` and leave it like this? It'll fix https://github.com/getsentry/sentry-javascript/issues/2943

Reply:
From what @HazAT has said, we ran into problems earlier in the summer when using `unknown` and had to switch back to `any`. Perhaps he can elaborate, but for now I'll put a pin in it and we can decide what to do later.